### ISSUE TITLE
Failed to get model_info when using scan function

### ISSUE THREAD

**Issue by ChristianBrowne** [butaixianran/Stable-Diffusion-Webui-Civitai-Helper#42]
**Having trouble using the scan function to create info files/previews for newly added Loras/hypernetworks/etc...

This issue only started happening today

Using the latest version installed from git url**

`Civitai Helper: Start scan_model
Civitai Helper: Scanning path: C:\Users\USERNAME\Documents\StableDiffusion\stable-diffusion-webui\embeddings
Civitai Helper: Checking preview image for model: C:\Users\USERNAME\Documents\StableDiffusion\stable-diffusion-webui\embeddings\bad-hands-5.pt
Civitai Helper: Model Info is empty
Civitai Helper: Creating model info for: cutev2-11000.pt
Civitai Helper: Calculate SHA256
Civitai Helper: Using Memory Optimized SHA256
Civitai Helper: sha256: 9bcbf6133c7e6067e2e517135c92762b384c1e214dde8a0565ed0e3da618f6ba
Civitai Helper: Request model info from civitai
Civitai Helper: Get error code: 403
Civitai Helper: <!DOCTYPE html>
<html lang="en-US">
<head>
    <title>Just a moment...</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <meta name="robots" content="noindex,nofollow">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <link href="/cdn-cgi/styles/challenges.css" rel="stylesheet">

</head>
<body class="no-js">
    <div class="main-wrapper" role="main">
    <div class="main-content">
        <h1 class="zone-name-title h1">
            <img class="heading-favicon" src="/favicon.ico" alt="Icon for civitai.com"
                 onerror="this.onerror=null;this.parentNode.removeChild(this)">
            civitai.com
        </h1>
        <h2 class="h2" id="challenge-running">
            Checking if the site connection is secure
        </h2>
        <noscript>
            <div id="challenge-error-title">
                <div class="h2">
                    <span class="icon-wrapper">
                        <div class="heading-icon warning-icon"></div>
                    </span>
                    <span id="challenge-error-text">
                        Enable JavaScript and cookies to continue
                    </span>
                </div>
            </div>
        </noscript>
        <div id="trk_jschal_js" style="display:none;background-image:url('/cdn-cgi/images/trace/managed/nojs/transparent.gif?ray=7a6df2372cd232e4')"></div>
        <div id="challenge-body-text" class="core-msg spacer">
            civitai.com needs to review the security of your connection before proceeding.
        </div>
        <form id="challenge-form" action="/api/v1/model-versions/by-hash/9bcbf6133c7e6067e2e517135c92762b384c1e214dde8a0565ed0e3da618f6ba?__cf_chl_f_tk=mDQRuDq.sRLKmrLYoq.uiofG33ty6_0KYkHb2pe7iBc-1678644125-0-gaNycGzNCXs" method="POST" enctype="application/x-www-form-urlencoded">
            <input type="hidden" name="md" value="hZnwVVuJQsirbd0fcvQ7N2gw1uacE_6b5MxDTJg7pWU-1678644125-0-AdLmb1lCClmllGZMIU1iSBsJlWVtajYBaTZIFaTymMujJLZEalA9Nw6Db9PHzCStTXIbIuU7nPSM-ePuVek8ageViGFSeB1W2KLQeAGOsdvieENPFIhuwUawjuRXZgMysr9ydkRtkP17k05gU_ajxp82-1rIMTfay-eKcZEp3GPV2ffw-T8VFGgL87DHrSKCcLPFleY9DIHcgXODhoA5jA7Y7pS7fBDDbRaJeKxsQx7an-54xNM7yalpPq_D9ONEG9XqeqNxIFLxZ3kPGBuu7aHvlfoVo44ZTEV4y7W5JDMoApUfI6FYt5KWN4ijTFgQIYTun6tAdIsPYGyDTypEH4OyrJMWK39kr9MrD0jekZIob4NWDA2Pwm6jaDH5v62xtvdZpRBMnHsUDGNFVQnFNr_tqUVMISHjas8kaWSiaJ9Jh0YJ19B-6YoMVzL_77YGRAwSUYynIUS9Ivfx7EYgcHtjg-0YIcJtnX8xZgh2nsfi4pRY3icVIXt3hjuX2ooVcgfVh1UHe_ZCa5BLldBtQwwx0d-vNRRlSe45WQWmhxZR6Qpx2-4suy9tn-KdOmJYA86HluwlfwP1HTh35g9ZdBG4nxpWkvzDD8jIGScS8MrxAtCx71HlUJvqkKZ5I1RExZ_u5kDEeYfoUN7jWH5AlfiuaK1gPqCFl4utf36CyToLX2J8S-kg3BwhjcOmxn1M3niqBGKrAaBZunpDRQXnUijvi_kPDLRxa97Yj5NweMiy29XYgJusLiV5jWl_Q9FlK-M9ZeGxT6t45dXdbWSVLZwPaMzZ__TShnBONqdXtsEJvQb_LL0MYcQwfh_txpEySh_P_eOI-xStkVa8R9SOVZoZXV1frKuuDCmEpk5IWkAnT-dw3IU6b4nlbxoh1FuS81o7ucJdS3ExwLn2HFqz8EFqj1waL0uXjfqHvg-E0raL5ZVePHcpftlQBIVbUiehsKUa2CZbzBNx27zlZT13ZAVtWRHm9APHUGUnfIq1_ZZrTmLqSV6nHbTs6jgcL6s_4h1wI1O2SNOwAUUnd4TybZuE6yC8SJqIKtXBUrTP_lJF63Cdo9QsBy_QsNPTn4pBhrk8aI29Z2TOZ191q3Z-DChcH1kbHK2WZJDEAMpBTzwCRipHQXVSzYcRE0z07Q-vsLhH_DOGt0NXetTOsYQHzYyzDGhYPL8l2FF4ZS9OY3xnoYzwLzjPhX1znVlRwmNcOeUb-a0w6mN7UYoHNlqkD253rt7gEDFtURW-5G5LY0IzIpEvCCB9hWwNMftk2gYDNaTs7bONJs2aoXKDZnS5FG2940cNUaBmIZbJW7JlTr-Fl5PYce8wnIO7-7AnEOFYJt-wogHfSizXM6Abo2_PyVCalRRUYJQ-i5OypdanXJKBGLmBHwPPGrcowVrymdJjeUn_awrFUvPAylxkZWh1klgGnTJvRyioIDqdvKcP01CGKKpQZjhP9rg32NG1U8ReMOmuTqJtpfdZfGtV_aqCKnyfBzoA4Bebnw5ohOLCQUuHLgy1LeY7_vDgJOYfztBDgfMqAUm-bvQziFMh40-oQc4L-2xjMfdIICInYS5pw01INosL9IBzo9Uvkb986kf6JyjIQFIoGyJLYKRqz0vcXTMvNs3k2G40L7ggKfBDnzkpQkmE7E7OfI56_nRDwj8DU48jwsVxNhAtbYuPK1ZLwodcPmdJSKbjGKudNcaMnXMqGz1aiZaXLfkuEc-gELh68XUE67DnFkTQjV6Nmy6HUCGJTs2dZ8v7UlayeYzDkezyn-8TiMkU79d74zEyLKIYWEXokdBZ6evlk5T5p84gBmAxxN2ZHroF5L1w3S8gZnOhtaQ3-At4AjQSD0oLj1lFRIeKGsVpdy0856Zdh4Z5h_mre6hEVVmnS3BNAOtaPNtzLU4j5dTHQvv5lTJ-DAGTwx3AlP66_WZIkN-TRK3WwJzk0w87VX8CgIMgRDXr9UxnMcOR-k80DEcnzK7MKPrf7Osdc5tE_eb_SDlpfVgwfFjXbyKyuBX-a3PoPvKUL5BRFzOteb_64myAgzd6c9f1y9pARs4I_9yrOH5xaBvOUzx6oueFVYghL1YU-4rpd3zJC7bIEP--tIqwXX-H2ouehXdWvxM_HAHaS5dxvLuMYBxbb8RDLiydeBgILeUoxaJ4x12xOs4Vnz6TdhpJWMwbvNvu5ayaXdrx9gFrhH6fCpU4B_es6xQfG6CRgP2Jfcv0J0lctVkGZnscR6-gG_QcG4ljqOUlKFsmpiMX_iQ5lr_mCJs5lRLwdAPdAz22umlzpeT-SflG3Yd8A3fJluTmzxKNnZDOKbypzpqoZ2ng2QjMZQdvj5yJgv8yIa9UPc1iwVAfeJ5XMc-g6rPXeEYCBA">
        </form>
    </div>
</div>
<script>
    (function(){
        window._cf_chl_opt={
            cvId: '2',
            cZone: 'civitai.com',
            cType: 'managed',
            cNounce: '82153',
            cRay: '7a6df2372cd232e4',
            cHash: '03e90ce05206ece',
            cUPMDTk: "\/api\/v1\/model-versions\/by-hash\/9bcbf6133c7e6067e2e517135c92762b384c1e214dde8a0565ed0e3da618f6ba?__cf_chl_tk=mDQRuDq.sRLKmrLYoq.uiofG33ty6_0KYkHb2pe7iBc-1678644125-0-gaNycGzNCXs",
            cFPWv: 'g',
            cTTimeMs: '1000',
            cMTimeMs: '0',
            cTplV: 4,
            cTplB: 'cf',
            cK: "",
            cRq: {
                ru: 'aHR0cHM6Ly9jaXZpdGFpLmNvbS9hcGkvdjEvbW9kZWwtdmVyc2lvbnMvYnktaGFzaC85YmNiZjYxMzNjN2U2MDY3ZTJlNTE3MTM1YzkyNzYyYjM4NGMxZTIxNGRkZThhMDU2NWVkMGUzZGE2MThmNmJh',
                ra: 'cHl0aG9uLXJlcXVlc3RzLzIuMjUuMQ==',
                rm: 'R0VU',
                d: 'MrZy004j1WM2vR9vCBD9Ndy63fTRiQ77aqLsfr63kzcXHurKp1LcV6NGND19Jbvwd8h0MiI07mtXVqvXj6d8v4VdySXl7l7q3FFgknaHaryk+pRi+4Um4t75wHzvqO0bD/U5Xja3f8YZp3xzFBVwG/FSfLfTyUWIhYDM6wLPfL4AcLbmPv9GrdxVYZwUTCMsXHkJ7sZTqBm5r3bSOLs7xlHH8fqf7Iu8UX7sq7jJCl3NGDptPv1Wlr0n/gVQUAgCM68Znez5mHMrXV8+tLVNVp6z5ForYqHn5OSKAl5o2beZjMDlsLflMX5PYii3jE0mmBQ3iDyYX8AiubZBAN+6t25YzGsZT5C0HC198pUpChmLqmhjCa41VFXovJXlAb97kyDwfpgCUisRaZk+FhrxkfhSCXcwYanXlQWZ9uTARknq4O3vGaIHGXXDZlknJyTd37VhdqSVOHJf68z0G7OWa08rI8t3q0unBYyCHGsTqVQXw+wHJSeQco0/2FWINcbVmod4Wnl4rPQ/7ewarw6x6wQ967PqNWlY+TQeoBgLIvhZ6StXrAcCgMFgzSIuG5ZnVndu+0SZ9ORapuT7MILb8fBRyjpGcVHTlZBgwfUA68pe84i1nx8K/9MFS9UmvqPuhdhG4NgbVZFJiC+TTiYctCAodD7aM/yW4tcYYY/d9N9+gsuWL90qZ+B0keDV/loV',
                t: 'MTY3ODY0NDEyNS4zMDMwMDA=',
                m: 'inXBp14UsFDrgH+3kUKb7GCWzhSXfRxsJlfu8kpnK2Q=',
                i1: '0eyh0nMtRd4JRzd0IL+qMg==',
                i2: 'WOpLGKU390v8d1CZX0tQGw==',
                zh: 'Uz+9QLA9AxaI0rN749mpcFD0OqOSQr34NUHtIEfAd74=',
                uh: '5GU+jYv2xJ+bCaE/ARmi/DORbiS/v56CW7E0TH4XWQk=',
                hh: 'Hd86XilH8WZpcWTr+6Ow5RAO2VpuQW9ZzMlqtxBBzFA=',
            }
        };
        var trkjs = document.createElement('img');
        trkjs.setAttribute('src', '/cdn-cgi/images/trace/managed/js/transparent.gif?ray=7a6df2372cd232e4');
        trkjs.setAttribute('alt', '');
        trkjs.setAttribute('style', 'display: none');
        document.body.appendChild(trkjs);
        var cpo = document.createElement('script');
        cpo.src = '/cdn-cgi/challenge-platform/h/g/orchestrate/managed/v1?ray=7a6df2372cd232e4';
        window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash;
        window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;
        if (window.history && window.history.replaceState) {
            var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;
            history.replaceState(null, null, "\/api\/v1\/model-versions\/by-hash\/9bcbf6133c7e6067e2e517135c92762b384c1e214dde8a0565ed0e3da618f6ba?__cf_chl_rt_tk=mDQRuDq.sRLKmrLYoq.uiofG33ty6_0KYkHb2pe7iBc-1678644125-0-gaNycGzNCXs" + window._cf_chl_opt.cOgUHash);
            cpo.onload = function() {
                history.replaceState(null, null, ogU);
            };
        }
        document.getElementsByTagName('head')[0].appendChild(cpo);
    }());
</script>

    <div class="footer" role="contentinfo">
        <div class="footer-inner">
            <div class="clearfix diagnostic-wrapper">
                <div class="ray-id">Ray ID: <code>7a6df2372cd232e4</code></div>
            </div>
            <div class="text-center" id="footer-text">Performance &amp; security by <a rel="noopener noreferrer" href="https://www.cloudflare.com?utm_source=challenge&utm_campaign=m" target="_blank">Cloudflare</a></div>
        </div>
    </div>
</body>
</html>

Civitai Helper: Failed to get model_info`

**Similar results happen when trying to load info from url**

`Civitai Helper: Run get_model_id_from_url
Civitai Helper: Run get_model_path_by_type_and_name
Civitai Helper: Request model info from civitai
Civitai Helper: Get error code: 403
Civitai Helper: <!DOCTYPE html>
<html lang="en-US">
<head>
    <title>Just a moment...</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <meta name="robots" content="noindex,nofollow">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <link href="/cdn-cgi/styles/challenges.css" rel="stylesheet">

</head>
<body class="no-js">
    <div class="main-wrapper" role="main">
    <div class="main-content">
        <h1 class="zone-name-title h1">
            <img class="heading-favicon" src="/favicon.ico" alt="Icon for civitai.com"
                 onerror="this.onerror=null;this.parentNode.removeChild(this)">
            civitai.com
        </h1>
        <h2 class="h2" id="challenge-running">
            Checking if the site connection is secure
        </h2>
        <noscript>
            <div id="challenge-error-title">
                <div class="h2">
                    <span class="icon-wrapper">
                        <div class="heading-icon warning-icon"></div>
                    </span>
                    <span id="challenge-error-text">
                        Enable JavaScript and cookies to continue
                    </span>
                </div>
            </div>
        </noscript>
        <div id="trk_jschal_js" style="display:none;background-image:url('/cdn-cgi/images/trace/managed/nojs/transparent.gif?ray=7a6e0253db3c334e')"></div>
        <div id="challenge-body-text" class="core-msg spacer">
            civitai.com needs to review the security of your connection before proceeding.
        </div>
        <form id="challenge-form" action="/api/v1/models/18696?__cf_chl_f_tk=.eYfl1WbVYDoQJ3yxZaZzbg9_xqTF_BMf1BVVZbgUGQ-1678644785-0-gaNycGzNCLs" method="POST" enctype="application/x-www-form-urlencoded">
            <input type="hidden" name="md" value="c7h2Ey4tKccczezem.sL5m4Z2TOs.11eAw0iL3xksqA-1678644785-0-ATv2FXD9y0wJMKUm7jF_JrIUGbXNfKwAdultgV_8AeHFVBDNoknhXsruUSCGfivEL94YnvxEW8ZNUqhiXB279MGmwV646FHhCaoEwnZJipllUfcZxTNCgh3FTWpUtwmrVs0NlwfTqoBcD69IUUvdCwu3HxMqDNKDyFJti5TyUdE_5bKL2_G8TQY9E5d98U86QXqjrvIfE_BqGN6BbrnIgVwiDaPJZF_HxhonGRT7n8oCiq0cwBS8se3PkapC9mnrzB3_pDuf8Pzvr4OVhOjTFmvGo6cVrqNX2nMo25lBY82bQVU4eBxXhRfx4QBegs9y3peVhMG9IfckNAPerYLcyXXL-XW2YvNofFz60dgdhFq3YvZ8hEsy1DD4abdxI-k3ldA1ub1xODiyBfOmln8EwDjhL3pNx6ybbvXWtfxiGClJbfySxWIhwrAgl4xDN9qROMDOc6wOP_LlVRn-oyOaRXFGbCrelXHX4fnFv-IKSMVrJAr4eqsErYSH02d8rig6M3DCvKF_Cxmf5IKCFloy4m-GTk6qeHQFZXlLBngP7db3Eo6Q95wSOA9Wifytg77s5O3_P5vXA3rPxOBYvbhlQ0mPywG_mcXOcELqDJ5pqxxgif0EMTu0SqYCbGOGlaaw0pNg9vwhdUbN2Zru9LjQDyXLSwUDDrl5ispvfPqgIFnu1gg6UqJmZ88MagL_R_wUm5XVTeITgMYYyG4WmTPaMIbeIHc1-ebpwhGEmRbUMj7Xp03xsU4eHYCBDs1ZXUmCjfihLUqt029J_cL4E3bVxZoWTxg7JbCgCHG2TNK27ntSlgdwstFpTzOlBIl_06X0EJE9IeawknhbhdTzLcOADn0TjG1rQBGU0XKNina6zwTMwV-gjFLTs2yTBXQgsW0ilGjGa3L6-XoA3fc_RwPVd1j1Bz6tXsJwKza1FG-TTozBI3peR2MrdFQHPK_Y_I35kXbOfjP8whr9KYzdOzkPQhXOE-sANJUGdl5pewoMCEYRsy0M260FTc2GNoX_DUuu5YCelx_ad4XAiyMQEiaFZGIovEpopi8Jr3d48zt4geyJPE_fnQw2DMVk2CcyEzJPAeuS0aQ51Y6EJUszwSZBVLQBapJGFDIfcgMgnGsR3Pe5iQH1cx2uujLNdZnLDyBrBH9ln42wi1JCxEP_Szhl2VJprRZGlDKdxZn_aPcs3vhLfjdN_fDqudM3Ao0_ewSBNOpvXXk9VjaCzr0XzzgOoRk2wFhu-bHTfOKLse34-Tugi8Ra-Y57f1nZvx6gSlbkz5W1vYsBBAkNfcosRr4BKBQJ0cijLh6XCz49AQsmLUbYt26epBBX7GjY-WxG-NSa9m5OpjJM46uMqGGIH7k4A4yQqYinCCF8qA0Vcctts9LluiktguHLJvTwSDuQXE4TF5Ia7YYdnHYTghctNZFa4Ku6_wxkFK3ky2nrU1gAuyhGOaLm5HPrLa4XWF0Ap4V2hhp_oU3ZHhMHo0o4HSOA-_7HdrfEGwsqYHLlJHH2DNDOBliwTouHnG1VRgyLrV-yCzNhwc67BZU7ePOdGXVM45lsfp7vuCSjmQfwcLEotBybLBCxwTXEyO7SR5B5ZAGZ4urUp4H39i5ivGxkoLpO1HyrhekdZlS1ty1R-MQ4tHGTO202IBH60U-58_eLfqkATEF6H4JZXkmrTv_ria4AhlTPiZ3J0UlPDRwe_w9yDeeRqjrCWDyUVh6HG4elG-aW2GpeTUMN2G6osa5ULZsjX5qpkovG32yXddYgT4YQ2qY-kpLgupghsGRFCUYS5GBD4zpZdyUb40UsD2Cqt-loUI2xIH5bdoSW2HI7HHdErBHem2StiWxmk5FrTUW3xsEFlCyr3ko84QISxFggNAdctdTtXgE6RXyE3kU17icd9i3DIRXysDZ2FLdqmVQzP59m5bUIf0MHvUORdkAAT2oUR1V4w-gD00uzV7WAyBl5_e4HdNB2Ru6D4w0_yElTBbN7C4q0g7JPwSlHi_LWqF-LIzHuCGUj9oGXt8k-spjktbQuIiyuwP0ukYatc1TCITltJMBZGVJ93hzQrn7hEFzDGnmElfyQ64kDru-5Cd1sj-cG1hY5mqVI6o10Ye0xJOMXN1gCrmQHs-FbTuAUrOCLbP1TqppRrixm5uES3J6zris-eobAwe_FDMPjvR6XKP3Oig">
        </form>
    </div>
</div>
<script>
    (function(){
        window._cf_chl_opt={
            cvId: '2',
            cZone: 'civitai.com',
            cType: 'managed',
            cNounce: '94677',
            cRay: '7a6e0253db3c334e',
            cHash: '53098c2f1986057',
            cUPMDTk: "\/api\/v1\/models\/18696?__cf_chl_tk=.eYfl1WbVYDoQJ3yxZaZzbg9_xqTF_BMf1BVVZbgUGQ-1678644785-0-gaNycGzNCLs",
            cFPWv: 'g',
            cTTimeMs: '1000',
            cMTimeMs: '0',
            cTplV: 4,
            cTplB: 'cf',
            cK: "",
            cRq: {
                ru: 'aHR0cHM6Ly9jaXZpdGFpLmNvbS9hcGkvdjEvbW9kZWxzLzE4Njk2',
                ra: 'cHl0aG9uLXJlcXVlc3RzLzIuMjUuMQ==',
                rm: 'R0VU',
                d: 'PcyNNWfLkS9yj2XIVhwJnBml7HGc2dpNulpkeUgaFmb+5EM+LOK1zOE2Mte7TEHenI1b3qD74z+vmXzHYXpQEqo94C4mbPo18tgRbQOl4FYu7zY2Ewt4JYPORKJih7AgbKu9/wctK4M2uWX+7pI9MoQp/voSLNTzdgteap3g6Zr4gsoxuk/gCprnVHzLrSCYXmRVOsjq9aiBjDn6DZS7i9SF95z21/K7cChDyQh7iN54Of+DXdGABb7NFkUj6gUlJL5XT/ayHpapHklNU9YJTCZFe7hfyN3VbqUQnciIu3YDHDePcT45jTWzBaZsMyXcrdsb4E0ErRWbcuK2L9A0OGCwNdhM73csmrvbs53DFjba9iIQJvTgvIx0ieuhpdMOlTs63o6QEJ0HR/h+dyc5mxobBHRi1k9dnz+txURPY49y0vaLqy6zMvRdQ+nJ4Gvi9lngZNnndD35/iozTOY7ACNWs6LuVRIRNnlGgSvlD5DI6IStyagKE5Dv6qvaQJN7FONJiMyiIOJkbiPyQVMILIFyTXLHm/k+ntKyxWAZVW57fYZb5XTsEOdVjEBb8YM3U0stU0PghQ6h/s4K2g29Vs+40yn5nBaNQBAJY8PRP8SnFi4wwCQsR1qui0F1ZmcTsnCGEHFnFGx7a1M8RLpza0NLJm6lb9T49r+MnNmpw0MMGAgLHm0X6joCwOBYwzP0',
                t: 'MTY3ODY0NDc4NS4yNTkwMDA=',
                m: 'hKgVgekvi6xFmpnJXqRnVPFfw+uOa13cSWvyX+4AmSg=',
                i1: 'RZayk+GxYN5FyBmoxGpbGQ==',
                i2: 'jG+l7pPKHfR4JVKckSw87Q==',
                zh: 'Uz+9QLA9AxaI0rN749mpcFD0OqOSQr34NUHtIEfAd74=',
                uh: '5GU+jYv2xJ+bCaE/ARmi/DORbiS/v56CW7E0TH4XWQk=',
                hh: 'Hd86XilH8WZpcWTr+6Ow5RAO2VpuQW9ZzMlqtxBBzFA=',
            }
        };
        var trkjs = document.createElement('img');
        trkjs.setAttribute('src', '/cdn-cgi/images/trace/managed/js/transparent.gif?ray=7a6e0253db3c334e');
        trkjs.setAttribute('alt', '');
        trkjs.setAttribute('style', 'display: none');
        document.body.appendChild(trkjs);
        var cpo = document.createElement('script');
        cpo.src = '/cdn-cgi/challenge-platform/h/g/orchestrate/managed/v1?ray=7a6e0253db3c334e';
        window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash;
        window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;
        if (window.history && window.history.replaceState) {
            var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;
            history.replaceState(null, null, "\/api\/v1\/models\/18696?__cf_chl_rt_tk=.eYfl1WbVYDoQJ3yxZaZzbg9_xqTF_BMf1BVVZbgUGQ-1678644785-0-gaNycGzNCLs" + window._cf_chl_opt.cOgUHash);
            cpo.onload = function() {
                history.replaceState(null, null, ogU);
            };
        }
        document.getElementsByTagName('head')[0].appendChild(cpo);
    }());
</script>

    <div class="footer" role="contentinfo">
        <div class="footer-inner">
            <div class="clearfix diagnostic-wrapper">
                <div class="ray-id">Ray ID: <code>7a6e0253db3c334e</code></div>
            </div>
            <div class="text-center" id="footer-text">Performance &amp; security by <a rel="noopener noreferrer" href="https://www.cloudflare.com?utm_source=challenge&utm_campaign=m" target="_blank">Cloudflare</a></div>
        </div>
    </div>
</body>
</html>

Civitai Helper: Failed to get model info by id: 18696
Civitai Helper: failed to get model info from url: https://civitai.com/models/18696/tae-takemi-or-persona-5
`

Let me know if there is any other info I can provide to help clarify things.
Thanks for the amazing work and great extension!

**Comment by butaixianran** [comment]
It already told you very clearly, it is you connection issue, not this extension's issue.

"civitai.com needs to review the security of your connection before proceeding."

**Comment by ChristianBrowne** [comment]
sorry i didn't know that was not just something it does everytime before launching
I'll look into resolving that

**Comment by butaixianran** [comment]
I'm guessing your scanning is too fast, so it send request to civitai too quickly, civitai treats this as an attack.

I add a quick fix to force scan wait 1 second for TI models. You can update it. You need to shutdown SD and re-launch after updateing this extension.

**Comment by ChristianBrowne** [comment]
No luck with the new update for the 1 second delay.
I'll keep working on determining if its a me problem or a civitai problem

Thanks for implementing a delay to help resolve it though!

**Comment by butaixianran** [comment]
have you read the document Like I mentioned above? Once your connection are treated as unsecure, you need to wait for a while, maybe hours. Keeps trying just keeps civitai blocking you out.

**Comment by ChristianBrowne** [comment]
I did read the document i just didn't mention that i was gonna wait a few hours and just do other things in the mean time

**Comment by humongous74** [comment]
I'm having the same problem here. So after a few hours (maybe more) then I will be able to do the scanning, correct?

**Comment by butaixianran** [comment]
ok, updated document for this. I am not sure, I can only guess since last website down of civitai, it modified its Cloudflare setting, to have a more strict rule for requests. 

For anyone who find this issue, update your extension to 1.5.1 or later. Then wait for Cloudflare unblocking you.

**Comment by ChristianBrowne** [comment]
Closing this thread as the latest update seems to have resolved my issue.

I tried the extension before updating 5 mins ago and got the same error message as before.
I then updated to the latest version using automatic1111's extension tab "check for updates" then pressed "apply and restart ui" then just to be extra sure closed my console window and reopened SD, clicked on "refresh Civitai helper" and then when i pressed "scan" in the civitai helper tab it worked with no problem.

Thank you for all the quick fixes, adjustments, and very fast response time. 

Again, this is a great extension and probably the most useful one i have tied with controlnets.
